app/service/update-appeal-service.ts

```diff
@@ -406,7 +408,10 @@ export default class UpdateAppealService {
       askForMoreTime: {
         ...(_.has(caseData, 'submitTimeExtensionReason')) && { reason: caseData.submitTimeExtensionReason },
         inFlight: hasInflightTimeExtension
-      }
+      },
+      listCmaHearingCentre: listCmaHearingCentre,
```

Comment:
do we want to add this to a property so it looks like the following?
```

{
  "hearing": {
    "hearingCentre": "MANCHESTER",
    "time": "12:05",
    "date": "01/01/2020"
  }
}
```

in the future we might need to add a flag to it .